### PR TITLE
[Feature #1] First feature was implemented 

### PR DESCRIPTION
**Pull request by khthana**
### Feature-1
**bold**
_Italic_

> Rose are red
`print('hello')`
[CE](www.ce.kmitl.ac.th)

- new-file was addded

1. new-file2 added

@thana-ho

### CODE REVIEW

File: new-feature.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+add new feature.txt
```
Comment (thana-ho) [inline new-feature.txt:1]: this line was good.

File: new-feature.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
```
Comment (thana-ho) [inline new-feature.txt:2]: Good

File: new-feature.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+modified
```
Comment (thana-ho) [inline new-feature.txt:3]: Excellent

**Review by thana-ho — COMMENTED** [review]
Edit and review again

**Comment by khthana** [comment]
แก้เสร็จแล้ว Merge เลยมัั้ย

**Comment by thana-ho** [comment]
OK

**Pull request merged**